src/utils/defaults.js

```diff
@@ -0,0 +1,8 @@
+module.exports.fetchOptions = {
+  timeout: 10000,
```

Comment:
try to retrieve a timeout also from `process.env.JTA_FETCH_TIMEOUT` and default to 10000 is missing

Reply:
done